README.rst

```diff
@@ -1,5 +1,10 @@
 
-# Anonlink Client
```

Comment:
What is the rational for changing from markdown to rst? I'm not opposed - anonlink is rst and clkhash is markdown so it would be impossible to be consistent :-)

Reply:
Oh I was adding the icon of codecov to readme and somehow the markdown file cannot render the icon properly so I changed it to rst.  But now I guess that rendering problem is related to codecov not markdown. I will double check and change it back to markdown

Reply:
Sweet - just go with your preference.